routes/songlist-db.js

```diff
@@ -251,8 +160,10 @@ router.route('/:song_id/edit')
         song.year = req.body.year
         song.lang = req.body.lang
         song.copyright = req.body.copyright
-        song.lyric = req.body.lyric
         song.contributor = req.body.contributor
+        var stringArr = (req.body.lyric || '').split(/\r?\n|\//)
+        song.lyric = undefined
```

Comment:
iwhy do you need to do this? setting it to undefined is a code smell (i.e. smells wierd)